encode.go

```diff
@@ -4,7 +4,7 @@ import (
 	"bytes"
 	"io"
 
-	"github.com/mattn/go-runewidth"
+	runewidth "github.com/mattn/go-runewidth"
```

Comment:
Please do not change this. The problem with the package name, and the inclusion of the alias is that the URL path doesn't match the path on disk and the declared package name in the actual `go-runewidth` code. Additionally, this is a cosmetic change not related to the rest of this PR.

Reply:
To clarify the above, "... the rationale for the inclusion of the package alias is that ... "